packages/website/blog/2024-04-30-announcing-typescript-eslint-v8-beta.md

```diff
@@ -0,0 +1,226 @@
+---
+authors:
+  - image_url: https://www.joshuakgoldberg.com/img/josh.jpg
+    name: Josh Goldberg
+    title: typescript-eslint Maintainer
+    url: https://github.com/JoshuaKGoldberg
+description: Announcing the release of typescript-eslint's v8 beta, including its changes and timeline.
+slug: announcing-typescript-eslint-v8-beta
+tags: [breaking changes, typescript-eslint, v7, v8]
+title: Announcing typescript-eslint v8 Beta
+---
+
+[typescript-eslint](https://typescript-eslint.io) is the tooling that enables standard JavaScript tools such as [ESLint](https://eslint.org) and [Prettier](https://prettier.io) to support TypeScript code.
+We've been working on a set of breaking changes and general features that we're excited to get in front of users soon.
+And now, we're excited to say that typescript-eslint v8 is ready for public beta testing! 🎉
+
+Our plan for typescript-eslint v8 is to:
+
+1. Have users try out betas starting in May of 2024
+2. Respond to user feedback for the next ~1-2 months
+3. Release a stable version summer of 2024
+
+Nothing mentioned in this blog post is set in stone.
+If you feel passionately about any of the choices we've made here -positively or negatively- then do let us know on [the typescript-eslint Discord](https://discord.gg/FSxKq8Tdyg)'s `#v8` channel!
+
+<!--truncate-->
+
+## Trying Out v8
+
+Please do try out the typescript-eslint v8 beta!
+
+### As A New User
+
+If you don't yet use typescript-eslint, you can go through our [configuration steps on the v8 _Getting Started_ docs](https://v8--typescript-eslint.netlify.app/getting-started).
+It'll walk you through setting up typescript-eslint in a project.
+
+To use v8 specifically, see the following section for an updated install command.
+
+### As An Existing User
+
+If you already use typescript-eslint, you'll need to first replace your package's previous versions of `@typescript-eslint/eslint-plugin` and `@typescript-eslint/parser` with `@rc-v8` versions:
+
+```shell
+npm i @typescript-eslint/eslint-plugin@rc-v8 @typescript-eslint/parser@rc-v8 --save-dev
```

Comment:
Question - should there be an alternate command for the @typescript-eslint/typescript-eslint setup?

"For projects using a flat config setup, `npm i @typescript-eslint/typescript-eslint@rc-v8 --save-dev`"

Reply:
Oh yes right I'll update this. It was a copy & paste from the old post.